Explorer/Assets/DCL/Utilities/NftUtils.cs

```diff
@@ -23,27 +24,36 @@ public static bool TryParseUrn(URN urn, out string contractAddress, out string t
                     return false;
                 urnSpan = urnSpan.Slice(DCL_URN_ID.Length + 1);
 
-                // TODO: allow 'matic' chain when Opensea implements its APIv2 "retrieve assets" endpoint
-                // (https://docs.opensea.io/v2.0/reference/api-overview) in the future
                 // 2: chain/network
-                var chainSpan = urnSpan.Slice(0, CHAIN_ETHEREUM.Length);
-                if (!chainSpan.Equals(CHAIN_ETHEREUM, StringComparison.Ordinal))
-                    return false;
+                var chainSpan = urnSpan.Slice(0, urnSpan.IndexOf(SEPARATOR));
                 urnSpan = urnSpan.Slice(chainSpan.Length + 1);
 
                 // 3: contract standard
                 var contractStandardSpan = urnSpan.Slice(0, urnSpan.IndexOf(SEPARATOR));
                 urnSpan = urnSpan.Slice(contractStandardSpan.Length + 1);
 
-                // 4: contract address
-                var contractAddressSpan = urnSpan.Slice(0, urnSpan.IndexOf(SEPARATOR));
-                urnSpan = urnSpan.Slice(contractAddressSpan.Length + 1);
+                // check if wearables is third-party
+                if (contractStandardSpan.ToString().Equals(COLLECTIONS_THIRDPARTY, StringComparison.Ordinal))
```

Comment:
Is there a better way here of utilizing the span to not call ToString everywhere and allocate memory each time we parse a URN? It could be a hotpath.

Reply:
You're completely right, there was a better way to do it without having to allocate new memory.
Changes done here: https://github.com/decentraland/unity-explorer/pull/1533/commits/70a2c6aa2bbe136c2a15b1c3c53487441f8c96c2